clarify "inside" term (#613)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/data-loss-prevention-policies.md

- Content contains a label. For more information, see the below section [Using a label as a condition in a DLP policy](#using-a-label-as-a-condition-in-a-dlp-policy).
    
- Content is shared with people outside or inside your organization.

> [!NOTE]

> > Users, who have non-guest accounts in host organization's Active Directory or Azure Active Directory tenant, are considered as people inside the organization.

Reviewer: ```suggestion
> Users who do not have guest accounts in a host organization's Active Directory or Azure Active Directory tenant are considered people inside the organization.
```
